README.md

```diff
@@ -46,23 +59,34 @@ Request metrics:
 # TYPE http_response_size_bytes summary
 ```
 
-## Building and Running
+## Building and running
+
+#### Building a binary
 ```
 go build fluentd-exporter.go
 ./fluentd-exporter
 ```
 
-## Grafana example
-The following [dashbaord](https://grafana.com/dashboards/3522) can be imported into grafana
-![Grafana dashboard](https://i.imgur.com/oBY6urR.png)
+#### Building a docker image
+```
+docker build -t fluentd_exporter .
+docker run -it fluentd_exporter go-wrapper run -scrape_uri <fluentd-metrics-endpoint>
```

Comment:
Hey @audip, the fluentd_exporter needs to be accessible via http for Prometheus to scrape. When running the container the 
```
-telemetry.address string
    	Address on which to expose metrics. (default ":9309") 
```
needs to be exposes.

I'd suggest changing this example so that is does something like:
```
docker run -it -p 9309:9309 fluentd_exporter go-wrapper run -scrape_uri <fluentd-metrics-endpoint>
```

Reply:
@V3ckt0r Absolutely, oversight on my part. I'll make that change